packages/core-snapshots/src/codecs/codec.ts

```diff
@@ -0,0 +1,125 @@
+import { Models } from "@arkecosystem/core-database";
+import { Container } from "@arkecosystem/core-kernel";
+import { Blocks, Interfaces, Transactions, Utils } from "@arkecosystem/crypto";
+import { decode, encode } from "msgpack-lite";
+import { camelizeKeys } from "xcase";
+
+import { Codec as ICodec } from "../contracts";
+import { Codec as CodecException } from "../exceptions";
+
+@Container.injectable()
+export class Codec implements ICodec {
```

Comment:
Rename this to `MessagePackCodec` as it isn't an abstract but a concrete implementation.

Reply:
Also, omit the `I` prefix from the interface that is implemented.